PixaPicker/PixaDataCoordinator.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  PixaDataCoordinator.swift
+//  PixaPicker
+//
+//  Created by Thomas Caraway on 11/22/19.
+//  Copyright © 2019 Thomas Caraway. All rights reserved.
+//
+
+import Foundation
+
+class PixaDataCoordinator {
+    
+    var delegate: PixaDataCoordinatorDelegate?
+    private var cellImageURLs = [URL]()
+    var imageCount: Int { return cellImageURLs.count}
+    var currentPageNumber = 1
```

Comment:
`currentPageNumber` should be `private(set) var` since users outside shouldn't be able to manipulate page value